### PR TITLE
feat: Quick start without VPC peering

### PR DESCRIPTION
**Pull request by TSowbaranika**
*Issue #, if available:*

*Description of changes:*

Template updated to Deploy MongoDB Atlas without VPC peering.

This PR is used to create cluster or the serverless Instances with the single stack.
So we have added serverless instance parameter based on the condition in the stack  template will create either cluster or serverless instance

Also actiavtes the required public resources.
**Stack created with sample Template**

Screenshot 2023-09-12 at 15 21 14

Screenshot 2023-09-12 at 15 21 42

**Response on the Atlas Screen for the reference:**

Screenshot 2023-09-12 at 15 27 05

Screenshot 2023-09-12 at 15 27 23

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: templates/mongodb-atlas-main.template.yaml [hunk 1]
```diff
@@ -176,20 +186,42 @@ Parameters:
...
+  CreateCluster: !Equals [!Ref InstanceType, 'Dedicated Cluster']
+  CreateServerless: !Equals [!Ref InstanceType, 'Serverless Instance']
 Resources:
   ActivateAtlasResources:
     Condition: ActivateResources
     Type: AWS::CloudFormation::Stack
     Properties:
       TemplateURL: !Sub
-        - 'https://${S3Bucket}.s3.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}templates/activate-mongodb-atlas-resources.template.yaml'
+        - 'https://mongodbtemplate.s3.us-west-2.amazonaws.com/stacktemplate/activate-mongodb-atlas-resources.template.yaml'
```
Comment (Adelmar92) [inline templates/mongodb-atlas-main.template.yaml:224]: here you left your s3, can we make it to the original?

**Comment by Adelmar92** [comment]
mybe we can remove the version 4.4 and add the 7.0 to the cluster version

**Review by Zuhairahmed — APPROVED** [review]

**Review by marcosuma — APPROVED** [review]

**Review by vsnyc — CHANGES_REQUESTED** [review]

File: templates/mongodb-atlas-main.template.yaml (hunk 1 above)
Comment (TSowbaranika) [reply templates/mongodb-atlas-main.template.yaml]: updated the default URL

File: templates/mongodb-atlas-main.template.yaml [hunk 2]
```diff
@@ -11,6 +11,12 @@ Metadata:
     Order: "1"
   AWS::CloudFormation::Interface:
     ParameterGroups:
+    - Label:
+        default: Instance to be created
```
[the remark this answers is not in the document]
Comment (TSowbaranika) [reply templates/mongodb-atlas-main.template.yaml]: updated

**Comment by vsnyc** [comment]
Could you please make the publisher ID a mapping, so it's easier to spot rather than it being inline in an ARN?

Here is an example: https://github.com/aws-quickstart/quickstart-tableau-server/blob/main/templates/tableau-server-linux.template.yaml#L262-L265
```
Mappings:
  DefaultConfiguration:
    MongoDB:
      PublisherID: bb989456c78c398a858fef18f2ca1bfc1fbba082
```

And then the `PublicTypeArn` can be:

```
PublicTypeArn: !Sub
- 'arn:aws:cloudformation:${Region}::type/resource/${publisher_id}/MongoDB-Atlas-ServerlessInstance'
- publisher_id: !FindInMap
    - DefaultConfiguration
    - MongoDB
    - PublisherID
```

**Review by Adelmar92 — APPROVED** [review]

**Review by vsnyc — APPROVED** [review]

**Pull request merged**